css/types/timing-function.json

```diff
@@ -0,0 +1,223 @@
+          }
+        },
+        "cubic_bezier": {
+          "__compat": {
+            "description": "<code>cubic-bezier()</code> with ordinate ∉ [0,1]",
+            "support": {
+              "webview_android": {
+                "version_added": true
+              },
+              "chrome": {
+                "version_added": "16"
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": "4"
+              },
+              "firefox_android": {
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "10"
+              },
+              "opera": {
+                "version_added": "12.1"
+              },
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": true
+              },
+              "safari_ios": {
+                "version_added": null
+              },
+              "samsunginternet_android": {
+                "version_added": null
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "steps": {
+          "__compat": {
+            "description": "<code>steps()</code>",
+            "support": {
+              "webview_android": {
+                "version_added": "4"
+              },
+              "chrome": {
+                "version_added": "8"
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": "4"
+              },
+              "firefox_android": {
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "10"
+              },
+              "opera": {
+                "version_added": "12.1"
+              },
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": "5.1"
+              },
+              "safari_ios": {
+                "version_added": "5"
+              },
+              "samsunginternet_android": {
+                "version_added": null
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "frames": {
```

Comment:
Unfortunately, I have to suggest dropping this entire `frames` section outright because of the outcome of the discussion mentioned in the notes. 😞 

My best understanding of the `frames()` saga is that:

1. Nobody ever shipped `frames()`
2. The CSS Working Group decided to change the keywords for `steps()` instead
3. Nobody has yet shipped the new `steps()` keywords

In other words, it's all been rather speculative, so there's no compatibility to speak of… yet.

Reply:
I think, you're right.